usersimcrs/user_modeling/preference_model.py

```diff
@@ -120,32 +106,14 @@ def get_item_preference(self, item_id: str) -> float:
             item_id: Item ID.
 
         Returns:
-            Randomly chosen preference, which is generally in [-1,1], but in
-            case of the simple preference model it's either -1 or +1.
+            Randomly chosen preference, which is generally in [-1,1].
```

Comment:
It doesn't have to be randomly chosen, right (since we're in the abstract class)?

Reply:
That's correct. I replaced by `Item preference, which is generally in [-1,1].`.